modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -3220,4 +3222,15 @@ protected static SnapshotFileTree snapshotFileTree(IgniteEx srv, String name) {
     protected static SnapshotFileTree snapshotFileTree(IgniteEx srv, String name, String path) {
         return new SnapshotFileTree(srv.context(), name, path);
     }
+
+    /**
+     * @param consistentId Consistent id of the node.
+     * @param nodeId Left node ID.
+     */
+    protected static void onNodeLeft(TcpCommunicationSpi spi, Object consistentId, UUID nodeId) {
```

Comment:
Let' add it to `CommunicationWorkerThreadUtils` - the class is already used in the refactored tests, and lays in the same package.